docs/Build.md

```diff
@@ -0,0 +1,72 @@
+Naggum Build Guide
+==================
+
+Windows
+-------
+
+To build Naggum under Windows, just use Visual Studio or MSBuild like that:
```

Comment:
I feel conflicted about "under" here. Google says that people say that a lot, but if you search for "on Windows", you get ten times more results. So that's just information; the decision is up to you.